pkg/kubelet/eviction/helpers.go

```diff
@@ -31,7 +31,7 @@ import (
 	statsapi "k8s.io/kubernetes/pkg/kubelet/apis/stats/v1alpha1"
 	evictionapi "k8s.io/kubernetes/pkg/kubelet/eviction/api"
 	kubetypes "k8s.io/kubernetes/pkg/kubelet/types"
-	schedulerutils "k8s.io/kubernetes/pkg/scheduler/util"
+	schedulerpodinfo "k8s.io/kubernetes/pkg/scheduler/internal/podinfo"
```

Comment:
This line should not compile because of having "internal" in the path. We will need to apply one of the three suggested fixed described in #71863 to resolve this.

Reply:
thank you, i have follow your advices to modify those.